opnsense/ifinfo/files/ifinfo.c

```diff
@@ -126,7 +126,7 @@ main(int argc, char **argv)
 				dname = NULL;
 			}
 		}
-		printit(&ifmd, dname);
+		printit(&ifmd, dname, i);
```

Comment:
Looks good in general, thanks! But is there no field in the interface data that holds this value to be 100% sure?

Reply:
I couldn't find a field in there, but we could use `if_nametoindex(ifmd.ifmd_name)` to get the interface index.
This however would be a but bit redundant IMHO, going from interface index (loop variable `i`) to its name (`ifmd.ifmd_name`) just to derive the interface index from it again, but I understand your point 🤔

Reply:
Looking at https://github.com/opnsense/src/blob/c163ff33fa8fc9295149981feb333f3ff44407a4/sys/net/if.c#L348 I don't think it will get any better than this so let's merge this :)

Reply:
👌🏻